### PR TITLE
[FAU-415] Update admission requirements filter

### PR DESCRIPTION
**Pull request by zhyian**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

**What is the current behavior?** (You can also link to an open issue here)
https://inpsyde.atlassian.net/browse/FAU-415

**What is the new behavior (if this is a feature change)?**
As was [discussed](https://inpsyde.slack.com/archives/C045KFJ9MB5/p1714722318584329?thread_ts=1714721099.554009&cid=C045KFJ9MB5), we extended the hardcoded options to what the client requires:

- “Restricted (NC)” / “Eingeschränkt (NC)”
- “Admission free” / “Zulassungsfrei”
- "Admission free with restriction” / “Zulassungsfrei mit Einschränkung”

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

**Other information**:
I believe the reason for the Psalm issues is that the [PR](https://github.com/RRZE-Webteam/FAU-Studium-Common/pull/3) hasn't been merged yet.

### CODE REVIEW

**Review by tyrann0us — APPROVED** [review]
Thank you for working on this! 🙏🏽

**Comment by tyrann0us** [comment]
Sorry, I forgot to mention that we should add an (untechnical) `CHANGELOG.md` entry.

**Review by amiut — APPROVED** [review]
Thanks for taking care of it, LGTM

**Review by shvlv — APPROVED** [review]
Thanks! Regarding the Common package, it's important not to forget to update `composer.lock` for both plugins after any Common changes (at least before release).

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Expanded admission requirements filter to include the option 'Admission free with restriction'.
```
Comment (tyrann0us) [inline CHANGELOG.md:13]:
```suggestion
- Expand the admission requirements filter to include the "Admission free with restriction" option.
```

**Pull request merged**